backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/entities/Shop.java

```diff
@@ -0,0 +1,34 @@
+package com.google.gpay.virtualqueue.backendservice.entities;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+import lombok.Data;
+
+@Data
+public class Shop {
+    private String shopkeeperUid;
```

Comment:
Let's rename this as shopId?

Reply:
This class already has a shopId. And this field is for shopkeeper's Uid. Resolved!